specs/engage-digital_postman2.json

```diff
@@ -10,6 +10,9 @@
       "item": [
         {
           "name": "Events",
+          "description": {
+            "type": "text/plain"
+          },
```

Comment:
hey @CoCraig I followed the instructions in https://github.com/ringcentral/engage-digital-api-docs/tree/master/specs and updated the postman file using swaggman, but somehow I ended up with all these changes. Do you know why? Maybe you updated swaggman?

Reply:
@CedricBm John recently updated the postman generation and it looks like it is now adding descriptions for each API added to Postman.  Descriptions were probably not there before which is why all these changes are occurring now.